netmanager-app/core/redux/device/selectors.ts

```diff
@@ -0,0 +1,27 @@
+// take a slice of the application state and return some data based on that
+import { useSelector } from 'react-redux';
+import { useAppSelector } from '../hooks';
+
+export const useDevicesData = () => {
+  return useAppSelector((state) => state.deviceRegistry.devices);
+};
+
+export const useMaintenanceLogsData = () => {
+  return useSelector((state) => state.deviceRegistry.maintenanceLogs);
+};
+
+export const useDeviceLogsData = (deviceName) => {
+  return useSelector((state) => state.deviceRegistry.maintenanceLogs[deviceName] || []);
+};
+
+export const useDeviceUpTimeData = (deviceName) => {
+  return useSelector((state) => state.deviceRegistry.upTime[deviceName] || {});
+};
+
+export const useDeviceBatteryVoltageData = (deviceName) => {
+  return useSelector((state) => state.deviceRegistry.batteryVoltage[deviceName] || {});
+};
+
+export const useDeviceSensorCorrelationData = (deviceName) => {
+  return useSelector((state) => state.deviceRegistry.sensorCorrelation[deviceName] || {});
+};
```

Comment:
Another copy and paste, follow the current coding style please which is apis => hooks => redux state => use in code. @danielmarv 
Refer to the sites pages. Also only add the code for what you are working on.